README.md

```diff
@@ -87,6 +100,8 @@ Flags:
       --prefix="openstack"       Prefix for metrics
       --endpoint-type="public"   openstack endpoint type to use (i.e: public, internal, admin)
       --collect-metric-time      time spent collecting each metric
+      --disable-slow-metrics     disable slow metrics for performance reasons
+      --all-clouds               Toggle the multiple cloud scrapping mode under /probe?cloud=
```

Comment:
If you don't mind my comment on this PR ... why not call the parameter "--multi-cloud" instead of "--all-clouds".
`all` somehow implies this causes all clouds to be returned in contract to a default. But that is not the case, it's just enabling support for "multi-cloud".

(Nitpick: it's `scraping`, not `scrapping`)

Reply:
@ichtar Apologies for not debating this earlier, but I tend to agree with @frittentheke's comment here. --multi-cloud sounds reasonable considering that it's the opposite to single cloud. I considered calling it --all-clouds as the verb/action of loading all clouds from the configuration, but multi-cloud encapsulates the feature's essence better. 

Also, the nitpick is a good finding,

Reply:
changed scrapping to scraping and all-clouds to multi-cloud